package.json

```diff
@@ -7,7 +7,7 @@
   "sideEffects": false,
   "repository": {
     "type": "git",
-    "url": "https://github.com/IBM/node-sdk-core.git"
+    "url": "git+https://github.com/IBM/node-sdk-core.git"
```

Comment:
This change was actually suggested in the warning messages displayed by `npm publish` in the build log of a recent main branch build :joy:
![image](https://github.com/user-attachments/assets/5829d7c4-7ef5-43b8-a4ef-26e1940f80a4)
Link: https://github.com/IBM/node-sdk-core/actions/runs/13032430882/job/36354639235
So... I ran `npm pkg fix` and it made the above change.

Reply:
It was just a coincidence that I was paying particularly close attention to the logs since I wanted to study the semantic-release messages :joy: